<?xml version="1.0"?>

<launch>

 <include file="/home/ros/catkin_ws/src/cv_camera/launch/cv_camera.launch" />

<node name="opencv_5" pkg="puissance4" type="vision" respawn="false" output="screen">
  </node>

  <node name="image_view_1" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/MT4/output_video_1"/>
    <param name="autosize" value="true" />
  </node>
    <node name="image_view_2" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/Red_By_Salmi_Manal/output_video_2"/>
    <param name="autosize" value="true" />
  </node>
    <node name="image_view_3" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/Yellow_By_Parc_Corentin/output_video_3"/>
    <param name="autosize" value="true" />
  </node>
  <node name="image_view_4" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/Mires_By_Salmi_Manal/output_video_4"/>
    <param name="autosize" value="true" />
  </node>
  
</launch>

